cmd/ocm/create/cluster/cmd.go

```diff
@@ -131,7 +137,7 @@ func init() {
 	fs.IntVar(
 		&args.computeNodes,
 		"compute-nodes",
-		4,
+		0,
```

Comment:
should be 2

Reply:
It's computed dynamically below, I just used 0 to avoid impression of fixed default.

But actually this sneaks in a behavior change outside --interactive mode, I'll split it to its own PR if we agree on behavior:
@igoihman @vkareh _Should `create cluster` without explicit --compute-nodes default to minimum according to `--ccs` & and `--multi-az`, or always 4 (which may be too little or too much)?_
(the smart default requires keeping `minComputeNodes()` logic of course)

- Reminder to self: setting 0 supressed Cobra's `(default 4)` suffix to the help text.  If we decide on smart default, that's good, but should say in our text that the default = minimum.